Library/Formula/teensy_loader_cli.rb

```diff
@@ -12,15 +11,34 @@ class TeensyLoaderCli < Formula
     sha256 "dcd10140babb4d2937ce376c89e9c24a2e8046d2cabdad2cfdbc2542afa14471" => :mavericks
   end
 
+  devel do
+    url "https://github.com/PaulStoffregen/teensy_loader_cli.git", :revision => "a5b2c6fdc14c3a1ce9f614c767d9c758b0a1ce96"
+    version "2.1.devel"
+
+    depends_on "libusb-compat"  => :optional
+  end
+
+  head do
+    url "https://github.com/PaulStoffregen/teensy_loader_cli.git"
+
+    depends_on "libusb-compat"  => :optional
+  end
+
   def install
     ENV["OS"] = "MACOSX"
-    ENV["SDK"] = MacOS.sdk_path || "/"
+
+    if build.with? "libusb-compat"
+      ENV["USE_LIBUSB"] = "YES"
+    else
+      ENV["SDK"] = MacOS.sdk_path || "/"
```

Comment:
`teensy_loader_cli` codebase is a bit outdated. There is a lot `#ifdef`s inside. Pure OSX codebase is not fully implemented. Using `libusb` allows use of Linux codepath which is fully implemented, especially allows for use of soft reboots of connected board.